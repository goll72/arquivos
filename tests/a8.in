4 ataque18.bin 1
1 country "BRAZIL"
