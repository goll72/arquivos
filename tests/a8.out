170516.640000
